cgmes-gl-server/src/main/java/com/powsybl/cgmes/gl/server/services/CgmesGlService.java

```diff
@@ -85,26 +84,16 @@ public void toGeoDataServer(String caseName, Set<Country> countries) {
         LOGGER.info("{} lines and {} substations are sent to the geodata server", lines.size(), substations.size());
     }
 
-    Network getNetwork(String caseName) {
-        CaseDataSourceClient caseServerDataSource = createCaseServerDataSource(checkCaseName(caseName));
-
+    Network getNetwork(UUID caseUuid) {
+        CaseDataSourceClient caseServerDataSource = createCaseServerDataSource(caseUuid);
         CgmesImport importer = new CgmesImport();
         Properties properties = new Properties();
         properties.put("iidm.import.cgmes.post-processors", "cgmesGLImport");
-        return importer.importData(caseServerDataSource, properties);
-    }
-
-    CaseDataSourceClient createCaseServerDataSource(String caseName) {
-        return new CaseDataSourceClient(caseServerBaseUri, caseName);
+        return importer.importData(caseServerDataSource, NetworkFactory.findDefault(), properties);
```

Comment:
Why did you add  NetworkFactory.findDefault() ? isn't already the default behaviour ?

Reply:
importData(datasource, properties) is now deprecated, that's why I changed it to this new version